watertap/core/membrane_channel0d.py

```diff
@@ -64,6 +65,32 @@ def add_geometry(
         Returns:
             None
         """
+        # volume will be added be calling add_geometry from ControlVolume0D
+        if not hasattr(self, "length") and not hasattr(self, "width"):
+            self._add_var_reference(length_var, "length", "length_var")
+            self._add_var_reference(width_var, "width", "width_var")
+
+        if hasattr(self, "length") and hasattr(self, "width"):
+            super().add_geometry()
+            units_meta = self.config.property_package.get_metadata().get_derived_units
+
+            if not hasattr(self, "channel_height"):
+                self.channel_height = Var(
+                    initialize=1e-3,
+                    bounds=(1e-4, 5e-3),
+                    domain=NonNegativeReals,
+                    units=units_meta("length"),
+                    doc="membrane-channel height",
+                )
+
+            # # TODO: negating spacer volume for now as it can be assumed negligible (Park et al., 2020: https://doi.org/10.1016/j.desal.2020.114625). Revisit.
```

Comment:
I don't think this is a valid assumption. The paper says "spacer volume is shown to be unimportant" but they are speaking in the context of SEC (0.5% increase). But this PR is trying to include volume for the purpose of dynamics which could be impacted by changes in channel volume. In the SI of this paper they state for their system, the spacers take up 10-20% of the channel volume. This could be significant.

Reply:
Yes, I noticed that and perhaps wasn't descriptive enough in my comment. I also agree with you.

However, since this PR is just a mini-step in the right direction (there will be subsequent mini-PRs to help bring in the dynamic capability for RO), I put this as to-do item to revisit later. 

Rather than accounting for the loss in volume attributed to spacer volume right now, which will be more important later when we're looking for accuracy, the focus right now is just making sure new additions don't break existing code and that the _functionality works_. I don't need spacer volume to test the functionality.

Reply:
Sounds good and makes sense to me!